CONTRIBUTING.md

```diff
@@ -0,0 +1,220 @@
+# Contributing
```

Comment:
This template might not be up to date. Maybe we can omit it, and decide what to use in the future?